.github/CHANGELOG.md

```diff
@@ -155,7 +155,8 @@
 1. [ADIRS] Wind, TAS, and GS indications depend on availability of IR and ADR data - @davidwalschots (David Walschots)
 1. [ADIRS] SAT, TAT, and ISA indications depend on the selected ADR (1 or 3) - @davidwalschots (David Walschots)
 1. [PRESS] First implementation of automatic pressurization system - @MJuhe (Miquel)
-2. [MISC] Added annunciator lights to interactive checklist - @oim1 (oim)
+1. [MISC] Added annunciator lights to interactive checklist - @oim1 (oim)
+1. [README] Changed wording under the installation heading for readability and simplicity - @scnichols (hearsay#6064)
```

Comment:
**thought**: The CHANGELOG is intended for changes affecting the aircraft, this isn't such a change.
**suggestion**: Remove the changelog entry.